docs/en/integrations/package-spec.asciidoc

```diff
@@ -45,7 +45,7 @@ The following assets are typically found in an Elastic package:
 * {kib}
 ** Dashboards
 ** Visualization
-** Index patterns
+** {Data-sources}
```

Comment:
This wasn't showing up as expected:

```suggestion
** {data-sources-cap}
```

<img width="285" alt="image" src="https://user-images.githubusercontent.com/10479155/159344884-e75bd328-449d-48e3-b7af-84eb7787b543.png">